cookbooks/aws-parallelcluster-slurm/libraries/update.rb

```diff
@@ -95,3 +95,43 @@ def is_login_nodes_removed?
   previous_config = YAML.safe_load(File.read(node['cluster']['previous_cluster_config_path']))
   previous_config.dig("LoginNodes") and !config.dig("LoginNodes")
 end
+
+def is_live_update_required?
```

Comment:
in future we can add unit tests for this specific function.

Reply:
Yes, tracked in non blocking gaps to address